yojson.opam

```diff
@@ -6,13 +6,13 @@ bug-reports: "https://github.com/ocaml-community/yojson/issues"
 dev-repo: "git+https://github.com/ocaml-community/yojson.git"
 doc: "https://ocaml-community.github.io/yojson/"
 build: [
-  ["jbuilder" "subst" "-n" name] {pinned}
-  ["jbuilder" "build" "-p" name "-j" jobs]
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
 ]
-run-test: ["jbuilder" "runtest" "-p" name]
+run-test: ["dune" "runtest" "-p" name]
```

Comment:
I think we'd better remove this until we have an actual test suite, same as for the `make` target.